use-shopping-cart/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "use-shopping-cart",
-  "version": "3.0.0-beta.15",
+  "version": "3.0.0-beta.16",
   "description": "Shopping cart state and logic for Stripe",
   "author": "dayhaysoos",
   "license": "MIT",
   "repository": "dayhaysoos/use-shopping-cart",
   "main": "dist/react.js",
-  "types": "./src/react.d.ts",
+  "types": "./dist/index.d.ts",
```

Comment:
This should end up being `./dist/react.d.ts` once we create all of the `.d.ts` files.

Reply:
This answers my other question, thank you. Although, why the react.d.ts instead of an index?

Reply:
@dayhaysoos Check out https://github.com/dayhaysoos/use-shopping-cart/pull/200#discussion_r632996943.

The `.d.ts` applies to the file its name matches. When your bundle is named `index.js` it makes sense to call it `index.d.ts` because it when you import `index.js` TypeScript and your editor will know to check `index.d.ts`. In our case, we have a lot of different bundles with different names. For example, `react.js` needs its own `react.d.ts` and `core.es.js` needs its own `core.es.d.ts`.